src/main/java/mekanism/common/FuelHandler.java

```diff
@@ -56,7 +58,7 @@ public FuelGas(int duration, double energyDensity)
 		public FuelGas(IFuel bcFuel)
 		{
 			burnTicks = bcFuel.getTotalBurningTime() / Fluid.BUCKET_VOLUME;
-			energyPerTick = bcFuel.getPowerPerCycle() * general.FROM_RF;
+			energyPerTick = bcFuel.getPowerPerCycle() * MjAPI.MJ;
```

Comment:
Nothing to comment on the values of BC ofcourse, but the conversion to Mekanism values might be wrong. I'll update the first post with the points that needs some checking.

To make things clear, nothing wrong with BC, my conversion of BC values to Mekanism values might be messed up as I don't know exactly what kind off values Mekanism uses internally.